libsplinter/src/network/peer_manager/peer_map.rs

```diff
@@ -151,6 +151,12 @@ impl PeerMap {
             .values()
             .find(|meta| meta.connection_id == connection_id)
     }
+
```

Comment:
Typo in comment "Add get_pending method to PeerMap " - "ha" -- also, no period after the last sentence.